content/javascript/concepts/promise/terms/all/all.md

```diff
@@ -56,6 +56,43 @@ Promise.all([promiseA, promiseB])
 The output would be:
 
 ```plaintext
-Results from Promise.all(): [144,42]
+Results from Promise.all(): [23,144]
 Operations for Promise.all() have finished.
 ```
+
+## Codebyte Example
+
```

Comment:
We need some introductory sentence for the codebyte that gives a brief overview of what it does.

Reply:
Thank you! I added an introductory sentence. I also clicked "Update branch" below, I'm not sure if I was supposed to do that :)

This my first time contributing to any open source project :D